src/context/todo.context.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Todo } from '../models/Todo';
+// eslint-disable-next-line import/no-cycle
```

Comment:
It's not good thing to disable linter especially for cycling imports.
You can do that right way: move all helpers which you want you use for both context and service to the separate file `helpers.ts`. So you avoid using cycle imports and your architecture will be cleaner.